android/settings.gradle.kts

```diff
@@ -28,6 +28,10 @@ dependencyResolutionManagement {
   repositories {
     google()
     mavenCentral()
+
+    // Snapshots:
+    maven("https://oss.sonatype.org/content/repositories/snapshots")
```

Comment:
We should use repository content filtering for snapshots, otherwise it can slow down dependency resolution as we need to check every dependency against every repo if not found.
I also think we should just use the s01 address and not both for the same performance reason.
```
content {
  includeGroup(“com.emergetools")
}
```

Reply:
Oh and I just learned that you can declare this as a snapshots only repository like so: https://docs.gradle.org/current/userguide/filtering_repository_content.html#maven_repository_filtering